bot/exts/info/information.py

```diff
@@ -523,6 +523,17 @@ async def json(self, ctx: Context, message: Message) -> None:
         """Shows information about the raw API response in a copy-pasteable Python format."""
         await self.send_raw_content(ctx, message, json=True)
 
+    async def _set_rules_command_help(self) -> None:
+        help_string = f"{self.rules.help}\n\n"
+        help_string += "__Available keywords per rule__:\n\n"
+
+        full_rules = await self.bot.api_client.get("rules", params={"link_format": "md"})
+
+        for index, (_, keywords) in enumerate(full_rules, start=1):
+            help_string += f"**Rule n°{index}**: {', '.join(keywords)}\n\n"
```

Comment:
I don't think we need the `n°` bit, just e.g. `Rule 1` would be fine as that's how we style it elsewhere (in the `#rules` channel on the server)

Reply:
Alrighty ! :D